app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    user.increment!(:posts_counter)
+  end
```

Comment:
- [ ] Kindly note that the `user` in this method is not defined because in your association you are referring to the author (`belongs_to :author, class_name: 'User'`). Kindly change use author instead of user in the `update_posts_counter` method. Hint :point_down: 

```rb
def update_posts_counter
    user.increment!(:posts_counter)
  end
```